### ISSUE TITLE
UPSTREAM Management

### ISSUE THREAD

**Issue by wzrdtales** [wzrdtales/nvm-ng#3]
Whenever you feel, upstream has notable changes, hit me up here and reference it and I will go ahead picking it up.

**Comment by newtmitch** [comment]
Hi @wzrdtales - came across your repo from a comment here: https://www.growingwiththeweb.com/2018/01/slow-nvm-init.html, which admittedly is a bit dated. I have a "loadnvm" solution that's less than optimal right now and was poking around to see if there are any better solutions at this point. Is your fork still reasonably operable since it's been a while since you've updated `master`? Are you using this solution still or have you moved on to something else?

**Comment by wzrdtales** [comment]
Yes, I am actively using my fork, with no problems at all. So it is maintained in the demand: I use it and around 50 fellow engineers and may be some other people from the internet.

I rarely watch upstream, since it is really just working perfectly fine, but if you're stumbling into problems solved in upstream I will take care to get them or also accept PRs of course.

**Comment by wzrdtales** [comment]
The upstream reference is btw in this branch : https://github.com/wzrdtales/nvm-ng/tree/hard-line

**Comment by wzrdtales** [comment]
And to add as a comment: Performance wise it still is in the same means superior as documented here already https://github.com/wzrdtales/nvm-ng/issues/2 and I would definitely notice since I am working solely in a tiling shell and nowhere else (neovim user). Just make sure to make use the fast-reuse flag to get the complete optimization.
